live-examples/html-examples/content-sectioning/css/header.css

```diff
@@ -0,0 +1,32 @@
+@import url('https://fonts.googleapis.com/css?family=Dancing+Script');
```

Comment:
Answered on Discourse. I feel that we should rather not for performance reasons to some degree, but also to keep the number of fonts we use low. It also makes review easier, because the fonts available has already been OK'd